### PR TITLE
Jason single entry modal

### PR DESCRIPTION
**Pull request by JasonMartin85**
Add single entry page

### CODE REVIEW

File: server/routes/sharepoint.route.js [hunk 1]
```diff
@@ -99,6 +99,7 @@ router.post("/lists/GetByTitle\\(':listTitle'\\)/items", (req, res) => {
 router.put(
   "/lists/GetByTitle\\(':listTitle'\\)/items\\(:itemId\\)",
   async (req, res) => {
+    console.log(req.body)
```
Comment (JacobTheEldest) [inline server/routes/sharepoint.route.js:102]: Please remove the console.log added to the endpoint.

File: client/src/Reservation/reservation.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  background: rgb(146, 63, 191);
+  opacity: .3;
+}
+.slick-center  {
+  background: rgb(255, 3, 154);
+  opacity: 1;
+}
+
+
+/* carousel slider style end */
```
Comment (JacobTheEldest) [inline client/src/Reservation/reservation.css:73]: Add newline

File: client/src/Reservation/Reservation.js [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
+  return (<>
+    <div></div>
+  </>);
 };
 
 export default Reservation;
+
+
+            // 
+            // <Slide index={2}>I am the third Slide.</Slide>
```
Comment (JacobTheEldest) [inline client/src/Reservation/Reservation.js:17]: Add newline

File: client/src/Asset/asset.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+@keyframes fadeInAnimation {
+  0% {
+      opacity: 0;
+  }
+  100% {
+      opacity: 1;
+   }
+}
+
+/* carousel slider style end */
```
Comment (JacobTheEldest) [inline client/src/Asset/asset.css:88]: Add newline

File: client/src/Asset/EditAsset.js [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+      </form>
+      <button className="text-black text=xl place-self-end absolute right-2 top-2" onClick={()=>setShowModal(false)}>❌</button>
+      </div>
+      </div>
+      </div>
+      </div>
+  </>)
+}
+
+export default EditAsset
```
Comment (JacobTheEldest) [inline client/src/Asset/EditAsset.js:148]: Add newline

File: client/src/Asset/AssetMap.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        <Controls>
+          <FullScreenControl />
+          <ZoomControl />
+        </Controls>
+      </Map>
+    </div>
+    : <></>}
+    </>);
+};
+export default AssetMap;
```
Comment (JacobTheEldest) [inline client/src/Asset/AssetMap.js:84]: Add newline

**Review by JacobTheEldest — CHANGES_REQUESTED** [review]

**Review by JacobTheEldest — APPROVED** [review]

**Review by ddbonilla — APPROVED** [review]

**Pull request merged**